.travis/build.sh

```diff
@@ -1,3 +1,35 @@
 #!/bin/sh
 curl -fsLO https://raw.githubusercontent.com/scijava/scijava-scripts/master/travis-build.sh
 sh travis-build.sh $encrypted_bfd4c2633768_key $encrypted_bfd4c2633768_iv
+
+if [ "$TRAVIS_SECURE_ENV_VARS" = true \
+	-a "$TRAVIS_PULL_REQUEST" = false \
+	-a -f "target/checkout/release.properties" ]
```

Comment:
Would it make sense to also deploy the binaries on the master branch, as SNAPSHOTs? Then people can easily test new builds of the launcher.

Reply:
Good idea! I am looking into it right now and will get back to you.